packages/devtools_flutter/lib/src/connect_screen.dart

```diff
@@ -98,9 +103,26 @@ class _ConnectScreenBodyState extends State<ConnectScreenBody> {
     );
   }
 
-  void connect() {
-    // TODO(https://github.com/flutter/devtools/issues/1112): Connect to a VM.
-    Navigator.pushNamed(context, '/connected');
+  Future<void> connect([_]) async {
+    final connected = await service.FrameworkCore.initVmService(
+          '',
+          explicitUri: Uri.parse(controller.text),
+        ).catchError((_) {
+          Scaffold.of(context).showSnackBar(
+            SnackBar(
+              content: Text(
+                'Unable to connect to Dart VM at "${controller.text}". '
+                'Please specify a running Dart VM URL.',
+              ),
+            ),
+          );
+        }) ??
+        false;
+    print('Connection status: $connected');
```

Comment:
this should be a log statement instead of a print.

Reply:
I've removed this.